.gitignore

```diff
@@ -0,0 +1 @@
+inventory
```

Comment:
This would only be relevant for development purposes. And actually, you can register the variable from the execution and continue? the inventry is only needed for targeting ansible on a different node.